src/fhir/timeline-event.ts

```diff
@@ -14,7 +14,7 @@ export function useTimelineEvents() {
 
   useEffect(() => {
     const models = compact(
-      flatten(concat(queries.map((query) => query.data?.map((m) => new TimelineEventModel(m)))))
+      flatten(concat(queries.map((query) => query.data.map((m) => new TimelineEventModel(m)))))
```

Comment:
I'm not sure that concat is needed here
```suggestion
      flatten(queries.map((query) => query.data.map((m) => new TimelineEventModel(m))))
```